bedrock/firefox/templates/firefox/new/protocol/base.html

```diff
@@ -8,6 +8,9 @@
 
 {% extends "firefox/base/base-protocol.html" %}
 
+{# FxA automated tests use this attribute to verify the page #}
+{% block html_attrs %}data-test-fxa-template="firefox-download-thanks"{% endblock %}
```

Comment:
I'm not sure we need this one as this base template is used now only for /firefox/windows/, /firefox/mac/ etc?